daemon/openrazer_daemon/dbus_services/dbus_methods/mamba.py

```diff
@@ -348,3 +348,54 @@ def get_supported_poll_rates(self):
     self.logger.debug("DBus call get_supported_poll_rates")
 
     return self.POLL_RATES
+
+
+@endpoint('razer.device.misc', 'setHyperPollingLED', in_sig='y')
+def set_hyperpolling_wireless_dongle_indicator_led_mode(self, mode):
+    """
+    Set the LED mode, takes in 1 char
+
+    :param mode: LED mode
+    :type mode: char
+    """
+    self.logger.debug("DBus call set_HyperPollingLED")
+
+    driver_path = self.get_driver_path('hyperpolling_wireless_dongle_indicator_led_mode')
+
+    # remember LED mode
+    self.ledmode = mode
+
+    with open(driver_path, 'w') as driver_file:
+        driver_file.write(str(mode))
+
+
+@endpoint('razer.device.misc', 'setHyperPollingPair', in_sig='s')
+def set_hyperpolling_wireless_dongle_pair(self, pid):
+    """
+    Set Pairing mode, takes in 1 string which is the PID
+
+    :param pid: product id
+    :type pid: char
+    """
+    self.logger.debug("DBus call set_HyperPollingPair")
+
+    driver_path = self.get_driver_path('hyperpolling_wireless_dongle_pair')
+
+    with open(driver_path, 'w') as driver_file:
+        driver_file.write(pid)
+
+
+@endpoint('razer.device.misc', 'setHyperPollingUnpair', in_sig='s')
+def set_hyperpolling_wireless_dongle_unpair(self, pid):
```

Comment:
Any way for something using this to know which pid is currently paired?

Reply:
I don't believe so, at least from the packet captures I've seen. I mentioned something to this effect in the original issue @ https://github.com/openrazer/openrazer/issues/1869

`Note that Synapse doesn't ask for the PID or mouse model, I believe it just tries to pair with the mouse that was previously connected wirelessly, or that is currently connected (wired) during the time of pairing. This is just a front-end convenience. If you enter an invalid PID, or try to pair when a mouse is already paired, the firmware will give up after 60 seconds and everything will be back to normal. If you were already paired prior to trying to break it, you will continue to be paired after the 60 second timeout and everything will continue to work. Synapse tries to shield you from these error conditions (e.g. by not letting you pair if the dongle is already paired), but again this is just a front end convenience and I haven't seen any harm in trying.`